src/libraries/System.Data.Odbc/src/System/Data/Odbc/System.Data.Odbc.notsupported.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Data.Odbc
```

Comment:
They were baselined out of the contract but exist in the implementation. we require this in order to satisfy the facades build

Reply:
What fails if we don't have them?  Certainly not shims build since that's expected to have missing types.

Reply:
missing types is expected for the types we get from the partial facades. i.e we dont generate the type forwards but this type is added manually here https://github.com/dotnet/runtime/blob/master/src/libraries/shims/manual/System.Data.forwards.cs#L10

Reply:
I see, because on Desktop the ODBC32 class is internal and thus GenFacades won't write the typeforward, however we needed it for binary serialization compat with desktop so we manually add it.

Rather than duplicating this type, let's split out its definition and share the source with the PNSE and the real build. 
https://github.com/dotnet/runtime/blob/8b50c519a8deaffed3f221874ecc39bc930a5573/src/libraries/System.Data.Odbc/src/System/Data/Odbc/Odbc32.cs#L126-L148

Make ODBC32 partial, move TypeForwardedFrom to partial file and include RETCODE.

Alternatively we could just add this type to the reference assembly, which you previously did.  I think either approach would be OK.